.travis.yml

```diff
@@ -11,14 +14,23 @@ install:
 
 script:
   - "pylint --msg-template='{path}:{line}: [{msg_id}({symbol}) {obj}] {msg}' epsagon/"
-  - tox
+  - pytest
   - npm install && export PATH=$(pwd)/node_modules/.bin:$PATH
-  - ./acceptance/run.sh $TRAVIS_BUILD_NUMBER
+  - ./acceptance/run.sh $TRAVIS_BUILD_NUMBER $TRAVIS_PYTHON_VERSION
 
+jobs:
+  include:
+    - stage: build-and-deploy
+      provider: script
+      python:
+        - "3.8"
```

Comment:
What's that used for?
It doesn't override the global Python version?

Reply:
yes, only for deploy